pkg/occlient/occlient.go

```diff
@@ -83,6 +83,11 @@ type Client struct {
 	namespace            string
 }
 
+type SvcCat struct {
```

Comment:
Please add comment explaining what this struct is

Reply:
Shouldn't this be in `service` package?

Reply:
`ServiceStatus` might be a better name

Reply:
if I put it in service pkg,

```
go build -o odo ./main.go
import cycle not allowed
package main
	imports github.com/redhat-developer/odo/cmd
	imports github.com/redhat-developer/odo/pkg/application
	imports github.com/redhat-developer/odo/pkg/occlient
	imports github.com/redhat-developer/odo/pkg/service
	imports github.com/redhat-developer/odo/pkg/occlient
```